redaxo/src/addons/debug/lib/debug_clockwork.php

```diff
@@ -71,4 +71,18 @@ public static function getStoragePath()
     {
         return rex_addon::get('debug')->getCachePath('clockwork.db');
     }
+
+    /**
+     * We cannot rely on rex::isDebugMode() because it is always true on the console.
+     * So we have to check the config file itself.
+     */
+    public static function isRexDebugEnabled(): bool
+    {
+        $coreConfigFile = rex_path::coreData('config.yml');
+        $coreConfig = rex_file::getConfig($coreConfigFile);
```

Comment:
Danke für den PR. Das entspricht meinem Vorschlag, allerdings hatte ich über die Konsequenzen nicht so nachgedacht. Das Yaml-Parsing ist recht "teuer". Und so haben wir jetzt bei jedem Aufruf das Yaml-Parsing.

Vielleicht nur in der Console den Wert selbst auslesen?
Und eigentlich können wir ihn auch aus der [Cache-Datei](https://github.com/redaxo/redaxo/blob/61a90a69feade3f376725e71def3530044c31660/redaxo/src/core/boot.php#L89) auslesen. Da die boot.php ja definitiv vorher kam, wissen wir eigentlich dass die Cache-Datei existieren und aktuell sein muss.

Reply:
daran musste ich auch denken, habe es als nicht so kritisch gesehen, da das debug addon ja eigentlich nicht für Prod-Systeme gedacht ist.

Aber wenn man den overhead so einfach minimieren kann, natürlich immer gerne.